packages/docgen/lib/markdown/formatter.js

```diff
@@ -103,54 +103,88 @@ module.exports = (
 			formatTag(
 				'Related',
 				getSymbolTagsByName( symbol, 'see', 'link' ),
-				( tag ) =>
-					`\n- ${ tag.name.trim() }${
-						tag.description ? ' ' : ''
-					}${ tag.description.trim() }`,
+				( tag ) => {
```

Comment:
Thanks for breaking out the conditional logic in the functions in this file. It improves the readability a lot.